tests/system/data_sources/test_bigquery.py

```diff
@@ -342,7 +341,7 @@
     "--config-file",
     CLI_CONFIG_FILE,
 ]
-EXPECTED_NUM_YAML_LINES_TIMESTAMP_MIN_MAX = 30
+EXPECTED_NUM_YAML_LINES_TIMESTAMP_MIN_MAX = 29
```

Comment:
Why did this number go down one?

Reply:
Since `calculated fields` is not appended in case of `validate column`
Previously, it was an empty list